jass/items/Spear Throwing And Abilites.j

```diff
@@ -108,10 +108,10 @@ library SpearThrowsAndAbilities initializer onInit requires PublicLibrary, DUMMY
         local SpearData data = GetTimerData(GetExpiredTimer())
         call ReleaseTimer(GetExpiredTimer())
         set dummy = masterCastAtCaster(data.caster, data.target, 0, 0, 'A0EH', "creepthunderbolt")
-        set SpearCastTable.integer_h[dummy] = data
-        if not dupPreventionTable.has_h(data.target) then
+        set SpearCastTable.integer[GetHandleId(dummy)] = data
+        if not dupPreventionTable.has(GetHandleId(data.target)) then
             call TriggerRegisterUnitEvent(theTrigger, data.target, EVENT_UNIT_DAMAGED)
-            set dupPreventionTable.boolean_h[data.target] = true
+            set dupPreventionTable.boolean[GetHandleId(data.target)] = true
         endif
         set dummy = null
     endfunction
```

Comment:
I know that you've upgraded table but can we please keep these functions? I found them much nicer to interact with/way less verbose. Should be easy to copy & paste the code from the old table to the new one.